packages/react-tinacms-inline/src/styles/focus-ring.tsx

```diff
@@ -19,7 +19,7 @@ import styled, { css } from 'styled-components'
 import { BlocksEmptyState } from '../blocks/inline-field-blocks'
 
 export interface FocusRingStyleProps {
-  offset?: number | { x: number; y: number }
+  offset?: number | { x: number; y: number } | undefined
```

Comment:
`undefined` is implied here by the optional `?`

Reply:
Yeah I can try changing this. I think I was getting a type error and this fixed it but it shouldn't have so I'll try reversing.